docs/docs/versions/overview.mdx

```diff
@@ -7,6 +7,10 @@ sidebar_label: Overview
 
 ## What’s new in LangChain?
 
+The following features have been added during the development of 0.2.x:
+
+- Rolled out `v2` of the [Event Streaming API](https://python.langchain.com/docs/expression_language/streaming/#using-stream-events)
```

Comment:
I think if we're adding this section we should align it with the features we publicize in blogs. It looks strange if this is the only bullet point IMO.